rhel8/profiles/cis.profile

```diff
@@ -248,7 +248,7 @@ selections:
     - security_patches_up_to_date
 
     ## 1.10 Ensure system-wide crypto policy is not legacy (Scored)
-    #- var_system_crypto_policy
+    - var_system_crypto_policy=fips
```

Comment:
If  `fips` crypto-policy is to be selected, I think rule `enable_fips_mode` should be added too.

I don't know what the profile should default to. `future` seems easier for people to adopt by default.

Reply:
I see. I have changed it to `future`